camelot/core/orm/entity.py

```diff
@@ -195,33 +195,8 @@ def __new__( cls, classname, bases, dict_ ):
                 if table is None or table.primary_key.issubset([]):
                     _class.id = schema.Column(PrimaryKey(), **options.DEFAULT_AUTO_PRIMARYKEY_KWARGS)
 
-        cls.register_class(cls, _class, dict_)
         return _class
 
-    def register_class(cls, _class, dict_):
```

Comment:
Ter info, deze werd niet meer gebruikt en had eigenlijk bij de vorige aanpassingen verwijderd mogen worden. Dit gedrag zit nu in de EntityFacadeMeta.